misc/userscripts/open_download

```diff
@@ -94,7 +95,7 @@ fi
 line=$(printf "%s\n" "${entries[@]}"  \
        | crop-first-column 55 \
        | column -s $'\t' -t \
-       | $ROFI_CMD "${rofi_default_args[@]}" $ROFI_ARGS) || true
+       | $ROFI_CMD "${rofi_default_args[@]}" "$ROFI_ARGS") || true
```

Comment:
One cannot put the whole thing in quotes, because this means that the shell will look for an executable file with the name `rofi -some -args` instead of `rofi` (and of course this would fail and wouldn't do anything).